src/sidebar/components/Annotation/Annotation.tsx

```diff
@@ -1,7 +1,8 @@
-import { Actions, Spinner } from '@hypothesis/frontend-shared';
+import { CardActions, Spinner } from '@hypothesis/frontend-shared/lib/next';
 import classnames from 'classnames';
 import { useMemo } from 'preact/hooks';
 
+import type { Annotation as TAnnotation } from '../../../types/api';
```

Comment:
Poke! We'll need a convention for this; I emulated how you did it when there was a reference conflict for `FeatureFlags` (`IFeatureFlags` in that case).

Reply:
I'm gonna use `I` and punt on this for the moment, because I believe the `Annotation` type needs work overall (to differentiate the true API annotation type vs. API annotation + client props, which is what the `Annotation` type is currently).